source/api/image/3.0/index.md

```diff
@@ -424,8 +424,8 @@ In order to support the above requirements, clients _SHOULD_ construct the image
 
 | Parameter | Canonical value |
 | --------- | --------------- |
-| region    | "full" if the whole image is requested, (including a "square" region of a square image)<br/>otherwise the `x,y,w,h` syntax. |
-| size      | "max" if the default size is requested,<br/>the `w,` syntax for images that should be scaled maintaining the aspect ratio,<br/>and the `w,h` syntax for explicit sizes that change the aspect ratio. |
+| region    | "full" if the whole image is requested<br/>otherwise the `x,y,w,h` syntax. |
+| size      | "max" if the default size is requested,<br/>otherwise the `w,h` syntax. |
```

Comment:
Is it accurate to say that `max` is the default size?

Reply:
agree, should say "if the maximum size is requested"